optaplanner-migration/src/main/resources/META-INF/rewrite/ToLatest9.yml

```diff
@@ -0,0 +1,2777 @@
+type: specs.openrewrite.org/v1beta/recipe
+name: org.optaplanner.migration.ToLatest9
+displayName: Upgrade to the latest OptaPlanner 9
+description: 'Upgrade to the latest OptaPlanner 9 and migrate to Jakarta packages.'
+recipeList:
+  - org.optaplanner.migration.ToLatest
+  - io.quarkus.updates.core.quarkus30.Quarkus3
+  - org.openrewrite.maven.UpgradeDependencyVersion:
+      groupId: org.optaplanner
+      artifactId: optaplanner-bom
+      newVersion: 9.x
+  - org.openrewrite.maven.UpgradeDependencyVersion:
+      groupId: org.optaplanner
+      artifactId: optaplanner-quarkus
+      newVersion: 9.x
+  - org.openrewrite.maven.UpgradeDependencyVersion:
+      groupId: org.optaplanner
+      artifactId: optaplanner-core
+      newVersion: 9.x
```

Comment:
These 3 artifacts are the main entry points with or without Quarkus. Not sure if we need to add the `optaplanner-spring-boot-starter` as well; in the Spring Boot quickstart, we import the BOM.

Reply:
We can always do it later, if we get feedback on the Beta.